firebase-functions/test/helper.ts

```diff
@@ -4,27 +4,31 @@ type CreateRequestOptions = {
   headers?: Record<string, unknown>;
   userPermissions?: Record<string, boolean> | null;
   body?: Record<string, unknown>;
+  params?: Record<string, unknown>;
+  userId?: string;
 };
 
 type CreateAdminOptions = {
   claimsSpy?: jest.SpyInstance;
   updateUserSpy?: jest.SpyInstance;
+  getUserSpy?: jest.SpyInstance;
   verifyIdTokenSpy?: jest.SpyInstance;
 };
 
 export function createRequest(options: CreateRequestOptions = {}): Request {
-  const { body, headers, userPermissions } = options;
+  const { body, headers, params, userPermissions, userId } = options;
```

Comment:
These look like they were in alphabetical order before this change.

Reply:
```suggestion
  const { body, headers, params, userId, userPermissions } = options;
```